base_layer/wallet_ffi/Cargo.toml

```diff
@@ -14,6 +14,7 @@ tari_p2p = { version = "^0.8", path = "../p2p" }
 tari_wallet = { version = "^0.8", path = "../wallet", features = ["test_harness", "c_integration"]}
 tari_shutdown = { version = "^0.8", path = "../../infrastructure/shutdown" }
 tari_utilities = "^0.3"
+tari_key_manager = { version = "^0.8", path = "../key_manager" }
```

Comment:
Is this supposed to be in this PR?

Reply:
It is fixing a compile error due to recent merging of development into this branch; was necessary after I rebased.